e2e/setup.go

```diff
@@ -93,10 +94,15 @@ type TestSuite struct {
 func CreateTestSuite(t *testing.T, testCfg *Cfg) (TestSuite, func()) {
 	ctx := context.Background()
 
-	// load signer key from environment
+	// load signer key from environment or generate a random one
 	pk := os.Getenv(privateKey)
 	if pk == "" && !testCfg.UseMemory {
-		t.Fatal("SIGNER_PRIVATE_KEY environment variable not set")
+		t.Logf("SIGNER_PRIVATE_KEY environment variable not set. Generated random private key")
+		randomBytes := make([]byte, 32)
+		if _, err := rand.Read(randomBytes); err != nil {
+			t.Fatalf("Failed to generate random private key: %v", err)
+		}
+		pk = hex.EncodeToString(randomBytes)
 	}
```

Comment:
What does generating a unique key do? Looking at the disperser server [implementation](https://github.com/Layr-Labs/eigenda/blob/master/disperser/apiserver/server.go), rate limits are only enforced when dispersing.

Reply:
Also authorization headers are only signed by the client when [dispersing](https://github.com/Layr-Labs/eigenda/blob/master/api/clients/disperser_client.go#L109-L202) as well

Reply:
All this to say - why can't the key just be a hard-set constant?